xamarin/Xamarin.Forms
# [iOS] Fix for issue 4295 text decorations gone wild after text update

### Description of Change ###

As described in issue #4295 text decorations get lost after the `Text` is changed on iOS. This PR contains a fix by simply also calling the `UpdateTextDecorations` when the text property is changed.

### Issues Resolved ### 


- fixes #4295

### API Changes ###

 
 None

### Platforms Affected ### 


- iOS

### Behavioral/Visual Changes ###
After updating to this fix, it be working!

### Before/After Screenshots ### 

Here is a (temp) reproduction in the gallery sample app, full video: https://www.dropbox.com/s/vfffvfupfjadlol/UnderlineFix.mov?dl=0

![underlinefix](https://user-images.githubusercontent.com/939291/48066509-182e4c00-e1ce-11e8-93a7-15e710732828.gif)

### Testing Procedure ###


See video above, simply create a `Label` with a text and text decoration, then change the text of the label. Before fix: text-decoration is gone, after fix: text-decoration stays.

### PR Checklist ###
